ui/app/templates/components/clients/usage-stats.hbs

```diff
@@ -46,5 +47,17 @@
         data-test-stat-text="non-entity-clients"
       />
     </div>
+    {{#if (gte @totalUsageCounts.secret_syncs 0)}}
```

Comment:
Out of curiosity - why did you use `gte` here and not just check for `@totalUsageCounts.secret_syncs` ?

Reply:
I assumed that we still want to show the column if `secret_syncs` are 0.

Reply:
Ah yeah, `StatText` handles 0 values, but I forgot ember templates would treat it as falsy